cmd/eval-symflower-codegen-testing/cmd/evaluate.go

```diff
@@ -69,10 +77,22 @@ func (command *Evaluate) Execute(args []string) (err error) {
 		for _, modelID := range command.Models {
 			model := model.Models[modelID]
 			language := language.Languages[languageID]
+			languagePath := filepath.Join(command.TestdataPath, language.ID())
 
-			if err := evaluate.EvaluateRepository(model, language, filepath.Join(command.TestdataPath, language.ID(), "plain")); err != nil {
-				log.Fatalf("%+v", err)
+			repositories, err := os.ReadDir(languagePath)
```

Comment:
Please do not change this part. Look at the comment `Check that models and languages can be evaluated by executing the "plain" repositories.` This execution should always be the "plain" repository, to check if the language and model and ... code are working at all.

Reply:
Since LLMs are non-deterministic we have a problem here with the benchmarks itself. Sometimes they come up with the correct solution, sometimes they don't. So a) the benchmark itself will be super flaky and b) testing a model with "plain" does not give us anything, i.e. the free Google model sometimes passes in the "plain" repository and sometimes does not.

Reply:
We need "some" test to make sure that execution is working. Since it sometimes works, and sometimes now. It seems to me that it is either a prompt problem or a temperature one.